README.md

```diff
@@ -1,25 +1,3 @@
 # Drone plugin for Helm 3
 
-TODO:
-
-* [x] Make a `.drone.yml` that's sufficient for building an image
-* [x] Make a `Dockerfile` that's sufficient for launching the built image
-* [x] Make `cmd/drone-helm/main.go` actually invoke `helm`
-* [x] Make `golint` part of the build process (and make it pass)
-* [x] Implement debug output
-* [x] Flesh out `helm upgrade` until it's capable of working
-* [ ] Implement config settings for `upgrade`
-* [ ] Implement `helm lint`
-* [ ] Implement `helm delete`
-* [ ] Look for command-line flags added in helm3; implement them
-* [ ] EKS support
-* [ ] Dotenv support
-* [ ] Example drone config in this README
-* [ ] Change `.drone.yml` to use a real docker registry
-
-Nice-to-haves:
-
-* [ ] Cleanup() method on Steps to close open filehandles, etc.
```

Comment:
WONTFIX: this would be a responsible practice on a persistent machine, but I don't think it's necessary in Drone's ephemeral containers.